src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -44,9 +93,21 @@ const eventStyleGetter = ({ resource }: Event) => {
 
 const CustomEvent = ({ title, event }: EventProps) => {
```

Comment:
Q: How are events added to the calendar through actions on the page? i.e. pressing `+` adds the event to the calendar

Reply:
When going to the time table, we will have the list from Manav's PR and that will be displayed on the right sidebar, and using that info we pass `calendarEvents` to the calendar and whenever that changes then the calendar will be updated.

Background: Events are added through passing in a list of `calendarEvents` to the `SchedulerCalendar` component. Each calendar event contains `MeetingTimes` which is Date - Date. Using `computeMeetingTimeDays` theres a handy dandy npm package called `RRule` that generates timestamps for each event and using these we make `events` that go in to the calendar.